payroll-processor-client/tsconfig.json

```diff
@@ -20,7 +25,7 @@
     ]
   },
   "angularCompilerOptions": {
-    "fullTemplateTypeCheck": true,
+    "strictTemplates": true,
```

Comment:
This is the new Angular 9 replacement for `fullTemplateTypeCheck`

https://angular.io/guide/template-typecheck#strict-mode